README.md

```diff
@@ -7,341 +7,171 @@ This library provides debugging functionality to Ruby (MRI) 2.6 and later.
 This debug.rb is replacement of traditional lib/debug.rb standard library which is implemented by `set_trace_func`.
 New debug.rb has several advantages:
 
-* Fast: No performance penalty on non-stepping mode and non-breakpoints.
-* [Remote debugging](#remote-debugging): Support remote debugging natively.
-  * [UNIX domain socket](/docs/remote_debugging.md#invoke-program-as-a-remote-debuggee)
-  * [TCP/IP](/docs/remote_debugging.md#tcpip)
-  * Integration with rich debugger frontend
+- Fast: No performance penalty on non-stepping mode and non-breakpoints.
+- [Remote debugging](#remote-debugging): Support remote debugging natively.
+  - [UNIX domain socket](/docs/remote_debugging.md#invoke-program-as-a-remote-debuggee)
+  - [TCP/IP](/docs/remote_debugging.md#tcpip)
+  - Integration with rich debugger frontend
     - [VSCode](/docs/remote_debugging.md#vscode) (or other DAP supporting clients)
     - [Chrome DevTools](/docs/remote_debugging.md#chrome-devtool-integration) (or other CDP supporting clients)
-* Extensible: application can introduce debugging support with several ways:
-  * By `rdbg` command
-  * By loading libraries with `-r` command line option
-  * By calling Ruby's method explicitly
-* Misc
-  * Support threads (almost done) and ractors (TODO).
-  * Support suspending and entering to the console debugging with `Ctrl-C` at most of timing.
-  * Show parameters on backtrace command.
-  * Support recording & reply debugging.
+- Flexible: Users can integrate with the debugger in multiple ways
```

Comment:
The grammar here is a bit odd, I think one of the following sounds better:

> Flexible: Users can integrate the debugger in multiple ways

or 

> Flexible: Users can use the debugger in multiple ways